youtube/local_playlist.py

```diff
@@ -52,7 +52,7 @@ def get_local_playlist_videos(name, offset=0, amount=50):
         try:
             info = json.loads(video_json)
             if info['id'] + ".jpg" in thumbnails:
-                info['thumbnail'] = "/youtube.com/data/playlist_thumbnails/" + name + "/" + info['id'] + ".jpg"
+                info['thumbnail'] = settings.img_prefix + "https://youtube.com/data/playlist_thumbnails/" + name + "/" + info['id'] + ".jpg"
```

Comment:
These should always be prefixed because they're requests for local files.

Reply:
What do you mean by "local files"? Are the thumbnails cached on disk?